erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md

```diff
@@ -0,0 +1,52 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
+
+### 1. Prerequisite
```

Comment:
```suggestion
To access the Invoice Discounting list, go to:
> Home > Accounting > Banking and Payments > Invoice Discounting

Prerequisites
```

Reply:
I didn't get you point...

Reply:
for mat is 1. How to create, so prerequisites shouldn't be numbered. ref: feedback from umair on another page